react/README.md

```diff
@@ -528,6 +534,51 @@ This guide provides a few guidelines on writing sensible React. Many of these ru
 
 
 
+## Modules
+
+- [7.1](#7.1) <a name="7.1"></a> When importing from React, explicitly import the exact features you need (in addition to React, which will be a signal to the compiler to compile JSX).
+
+  > Why? Named imports explicitly indicate what you will be using at the top of the file, and reduce the duplication of accessing properties on `React`.
+
+  ```js
+  // bad
+  import React from 'react';
```

Comment:
:-1:, when React inevitably does an ES6 version we will be happy that we are only importing what we need so that we can tree shake.